src/App.tsx

```diff
@@ -1,73 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
-import { peopleFromServer } from './data/people';
+import { Dropdown } from './component/Dropdown';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] =
+    useState<string>('No selected person');
```

Comment:
```suggestion
  const [selectedPerson, setSelectedPerson] =
    useState<Person | null>(null);
```
You should store the person instead of string

Reply:
> You should store the person instead of string

Not fixed